AppService/templates/linux/asp.net-core-webapp-on-azure.yml

```diff
@@ -0,0 +1,31 @@
+# Docs for the Azure Web Apps Deploy action: https://github.com/Azure/webapps-deploy
+# More GitHub Actions for Azure: https://github.com/Azure/actions
+
+name: Build and deploy ASP.Net Core app to Azure Web App
+
+on: [push, pull_request]
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@master
+    
+    - name: Set up .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: '<dotnet-core-version>'
+
+    - name: Build with dotnet
+      run: dotnet build --configuration Release
+    
+    - name: dotnet publish
+      run: dotnet publish -c Release -o ${{env.DOTNET_ROOT}}/myapp 
+
+    - name: Deploy to Azure Web App
+      uses: azure/webapps-deploy@v1
+      with: 
+        app-name: '<web-app-name>'
```

Comment:
I'm a little worried about using variable names that have "<variableName>" in it.  @nertim was suggesting we use ${variableName} instead.

Reply:
That's fine by me, we just need to be careful to skip any instances of GitHub's double-curly syntax for secrets and environment variables (`${{ some_value }}`) .

So instead of this: 
`publish-profile: ${{ secrets.<AZURE_WEBAPP_PUBLISH_PROFILE>}}` 
it would be:
`publish-profile: ${{ secrets.${AZURE_WEBAPP_PUBLISH_PROFILE} }}`

Reply:
I think doing the replacement properly should be fine if we just do it with a regex.  That said, it might be hard to read so I'll let @nertim decide if he wants to go with a different syntax.  [This page](https://en.wikipedia.org/wiki/String_interpolation) has a lot of good examples for different string interpolation syntax.  In any case for now I think let's just move forward with the single curly brace syntax since that's what @nertim originally wanted.

Reply:
Sounds good! Just made the conversion